src/views/flow_viewer_details_vw.sql

```diff
@@ -0,0 +1,139 @@
+and    iav.prov_var_type = 'JSON'
+join   json_table (iav.prov_var_json, '$[*]' 
+       columns 
+           ( Iteration_path varchar2(50) path '$.iterationPath',
+             status         varchar2(20) path '$.status',
+             loopCounter    number       path '$.loopCounter',
+             description    varchar2(80) path '$.description',
+             stepKey        varchar2(20) path '$.stepKey'
+           )) iter
+   on  1=1
+--UNION ALL
+--select prcs_id as prcs_id
+--     , null, null, null, null, null, null, 'Main', null, prcs_status
+--  from flow_processes 
+) 
+  select prcs.prcs_id
+       , prcs.prcs_name
+       , prdg.prdg_id
+       , prdg.prdg_prdg_id
+       , prdg.prdg_diagram_level as diagram_level
+       , ip.iter_path
+       , prdg.prdg_calling_dgrm as calling_dgrm
+       , prdg.prdg_calling_objt as calling_objt
+       , dgrm.dgrm_name || case when prdg.prdg_diagram_level is not null then ' ( Level: ' || prdg.prdg_diagram_level || ' )' else '' end as breadcrumb
+       , 1 as drilldown_allowed
+       , dgrm.dgrm_id
+       , dgrm.dgrm_name
+       , dgrm.dgrm_version
+       , dgrm.dgrm_status
+       , dgrm.dgrm_category
+--       , dgrm.dgrm_content
+       , ( select acomp.bpmn_ids
+             from all_completed acomp
+            where acomp.prcs_id = prcs.prcs_id
+              and acomp.dgrm_id = dgrm.dgrm_id
+              and acomp.diagram_level =  prdg.prdg_diagram_level
+              and ((acomp.iter_path = ip.iter_path and ip.iter_path is not null ) or (ip.iter_path is null and acomp.iter_path is null))
+         ) as all_completed
+       , null as last_completed     -- remove in v22.1
+       , ( select acurr.bpmn_ids
+             from all_current acurr
+            where acurr.prcs_id = prcs.prcs_id
+              and acurr.dgrm_id = dgrm.dgrm_id
+              and acurr.diagram_level =  prdg.prdg_diagram_level
+              and ((acurr.iter_path = ip.iter_path and ip.iter_path is not null ) or (ip.iter_path is null and acurr.iter_path is null))
+         ) as all_current
+       , ( select aerr.bpmn_ids
+             from all_errors aerr
+            where aerr.prcs_id = prcs.prcs_id
+              and aerr.dgrm_id = dgrm.dgrm_id
+              and aerr.diagram_level =  prdg.prdg_diagram_level
+              and ((aerr.iter_path = ip.iter_path and ip.iter_path is not null ) or (ip.iter_path is null and aerr.iter_path is null))
+         ) as all_errors
+       , prov.prov_var_vc2 as prcs_business_ref
+    from flow_processes prcs
+    join flow_instance_diagrams prdg
+      on prdg.prdg_prcs_id = prcs.prcs_id
+    join flow_diagrams dgrm
+      on prdg.prdg_dgrm_id = dgrm.dgrm_id
+    join iteration_paths ip
+      on ip.prcs_id = prcs.prcs_id
+    left join flow_process_variables prov
+      on prov.prov_prcs_id  = prcs.prcs_id
+     and prov.prov_var_name = 'BUSINESS_REF'
+     and prov.prov_var_type = 'VARCHAR2'
+     and prov.prov_scope    = 0
+     where prcs.prcs_id = 8416;
```

Comment:
process id included in the view.

Reply:
This file should be deleted - we don't use it.  (was an idea we tested and decided not to use - the viewer is still driven from flow_instance_details_vw, as before.   It now has 2 versions, one simple one in CE and a more complex one in EE - d we fixed the file locations / names yesterday...